dataset/src/main/scala/frameless/TypedEncoder.scala

```diff
@@ -34,16 +35,19 @@ object TypedEncoder {
     def nullable: Boolean = true
 
     def sourceDataType: DataType = ScalaReflection.dataTypeFor[Unit]
+
```

Comment:
I don't want to start a holy war, but I don't like these line breaks and they weren't here before. However, we don't have any guidelines for code style and formatting, so the only way I can argue here is minimizing the diff and keeping code consistent.